examples/3d/render_to_texture.rs

```diff
@@ -0,0 +1,216 @@
+        transform.rotation *= Quat::from_rotation_y(0.7 * time.delta_seconds());
+    }
+}
+
+fn setup(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+    mut active_cameras: ResMut<ActiveCameras>,
+    mut images: ResMut<Assets<Image>>,
+    mut clear_color: ResMut<ClearColor>,
+) {
+    let size = Extent3d {
+        width: 512,
+        height: 512,
+        ..Default::default()
+    };
+    let image = Image {
+        data: vec![0; size.width as usize * size.height as usize * 4],
+        texture_descriptor: TextureDescriptor {
+            label: None,
+            size,
+            dimension: TextureDimension::D2,
+            format: TextureFormat::Bgra8UnormSrgb,
+            mip_level_count: 1,
+            sample_count: 1,
+            usage: TextureUsages::TEXTURE_BINDING
+                | TextureUsages::COPY_DST
+                | TextureUsages::RENDER_ATTACHMENT,
+        },
+        ..Default::default()
+    };
+    let image_handle = images.set(RENDER_IMAGE_HANDLE, image);
+
+    let cube_handle = meshes.add(Mesh::from(shape::Cube { size: 4.0 }));
+    let cube_material_handle = materials.add(StandardMaterial {
+        base_color: Color::rgb(0.8, 0.7, 0.6),
+        reflectance: 0.02,
+        unlit: false,
+        ..Default::default()
+    });
+
+    commands
+        .spawn_bundle(PbrBundle {
+            mesh: cube_handle,
+            material: cube_material_handle,
+            transform: Transform::from_translation(Vec3::new(0.0, 0.0, 1.0)),
+            ..Default::default()
+        })
+        .insert(FirstPassCube)
+        .insert(RenderLayers::layer(1));
+
+    // light
+    // note: currently lights are shared between passes!
+    commands.spawn_bundle(PointLightBundle {
+        transform: Transform::from_translation(Vec3::new(0.0, 0.0, 10.0)),
+        ..Default::default()
+    });
+
+    // camera
+    let render_target = RenderTarget::Image(image_handle);
+    clear_color.insert(render_target.clone(), Color::WHITE);
+    let mut first_pass_camera = PerspectiveCameraBundle {
+        camera: Camera {
+            name: Some(FIRST_PASS_CAMERA.to_string()),
+            target: render_target,
+            ..Default::default()
+        },
+        transform: Transform::from_translation(Vec3::new(0.0, 0.0, 15.0))
+            .looking_at(Vec3::default(), Vec3::Y),
+        ..Default::default()
+    };
+    active_cameras.add(FIRST_PASS_CAMERA);
+
+    let camera_projection = &mut first_pass_camera.perspective_projection;
+    camera_projection.update(size.width as f32, size.height as f32);
+    first_pass_camera.camera.projection_matrix = camera_projection.get_projection_matrix();
+    first_pass_camera.camera.depth_calculation = camera_projection.depth_calculation();
+
+    commands
+        .spawn_bundle(first_pass_camera)
+        .insert(RenderLayers::layer(1));
```

Comment:
Should this be part of our camera bundles by default, or is it only required when we have multiple cameras?

Reply:
When there is no `RenderLayers` component (for any entity), the default of `RenderLayers::layer(0)` is used. I do think `RenderLayers::all()` would be a more intuitive default, but I have not modified the existing behavior - that should be a separate PR if desired.